File: year_1/PROGKI/Problem set 1/mario.c
/*
 * mario.c
 *
 * Introduction Programming Artificial Intelligence
 * Tim Stolp
 *
 * Asks for height.
 * Prints pyramide.
 */

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Asks for valid height number.
    int height;
    do
    {
        printf("Height: ");
        height = get_int();
    }
    while(height < 0 || height > 23);

    // Prints out pyramide.
    for(int i = 0; i < height; i++)
    {
        for(int k = 0; k < height - (i + 1); k++)
        {
            printf(" ");
        }

        for(int k = i + 1; k > 0; k--)
        {
            printf("#");
        }
        printf("  ");

        for(int k = i + 1; k > 0; k--)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: year_1/PROGKI/Problem_Set_4/mario.py
#
# mario.py
#
# Artificial Intelligence
# Tim Stolp
#
# draw a pyramide of a certain height
#
#####################################

# ask for height untill a value between 0 and 23 is put in
while True:
    h = int(input("Height: "))
    if h > -1 and h < 24:
        break

# print h amount of layers starting 2x# adding 1 # each next layer
for i in range(2, h + 2):
    print("{:>{height}}".format("#" * i,  height = str(h + 1)))
